Code/Editor/main.cpp

```diff
@@ -11,6 +11,7 @@
 
 int main(int argc, char* argv[])
 {
+    const AZ::Debug::Trace tracer;
```

Comment:
const? It is constant?

Reply:
Sure. I mean, nothing in this function even references it. The constructor keeps a non-const reference to it though...